frontend/src/metabase/parameters/components/widgets/ParameterFieldWidget.jsx

```diff
@@ -138,42 +157,63 @@ export default class ParameterFieldWidget extends Component<*, Props, State> {
           hasArrow={false}
           onClose={() => focusChanged(false)}
         >
-          <FieldValuesWidget
-            value={unsavedValue}
-            parameter={this.props.parameter}
-            parameters={this.props.parameters}
-            dashboard={this.props.dashboard}
-            onChange={value => {
-              this.setState({ value });
-            }}
-            placeholder={placeholder}
-            fields={fields}
-            multi
-            autoFocus
-            color="brand"
-            style={{
-              borderWidth: BORDER_WIDTH,
-              minWidth: this.state.widgetWidth
-                ? this.state.widgetWidth + BORDER_WIDTH * 2
-                : null,
-            }}
-            className="border-bottom"
-            minWidth={400}
-            maxWidth={400}
-          />
-          {/* border between input and footer comes from border-bottom on FieldValuesWidget */}
-          <div className="flex p1">
-            <Button
-              primary
-              className="ml-auto"
-              disabled={savedValue.length === 0 && unsavedValue.length === 0}
-              onClick={() => {
-                setValue(unsavedValue.length > 0 ? unsavedValue : null);
-                focusChanged(false);
-              }}
-            >
-              {savedValue.length > 0 ? "Update filter" : "Add filter"}
-            </Button>
+          <div className={cx(!isEqualsOp && "p2")}>
+            {verboseName && !isEqualsOp && (
+              <div className="text-bold mb1">{verboseName}...</div>
+            )}
+
+            {_.times(numFields, index => {
```

Comment:
This code resembles `DefaultPicker` quite a bit. Eventually, I hope to reconcile differences between the two implementations but that'll take some additional refactoring of those components, as they expect a card/question instance, which we don't have here.